friends/src/Components/Friends.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { getFriends } from '../actions';
+import Friend from './Friend'
+
+class Friends extends React.Component {
+    constructor() {
+        super();
+    }
+
+    componentDidMount() {
+        this.props.getFriends();
+    }
+    // state = {
+    //     newFriend: ''
+    // }
+
+    // handleChanges = e =>{
+    //     this.setState({newFriend: e.target.value});
+    // };
+
+    // addMember = e => {
+    //     e.preventDefault();
+    //     this.props.addMember(this.state.newFriend);
+    //     this.setState({ newFriend: ''})
+    // };
+
+    render() {
+        console.log(this.props.friends)
+        return (
+            <div className='friends'>
+                <h1>this is the friends page</h1>
+                {/* {this.props.friends.map(friend =>{
+                        return <Friend key = {friend.name} friend = {friend}/>
+                    }
+                )} */}
```

Comment:
You could render some JSX here as a fallback while the `friend` objects are being retrieved from the server (make a condition using the `this.props.isLoading` variable from `mapStateToProps`).